scripts/aggregate-themes/README.md

```diff
@@ -18,22 +17,27 @@ project
 
 Within this file declare and export an object containing your theme configuration.
 
-This configuration will be used to aggregate nested dependency themes and output a single `aggregated-themes.js` file. This file will be automatically included as an entry point within your application if you are using the webpack configuration provided by terra-toolkit.
+Theme aggregation uses this configuration to search for theme files within the following directory format: `theme->themeName->themeName.scss`. Then, for each theme, a root and/or scope themes will be generated.
 
-Theme files must follow naming conventions to be aggregated. Theme files are expected be within a namespaced directory within a `themes` directory.
+The generated file(s) output to the `generatedThemes` directory and are imported into `aggregated-themes.js`. This js file will be automatically included as an entry point within your application if you are using the webpack configuration provided by terra-toolkit.
```

Comment:
Should the generatedThemes directory be camelCase or hyphen case? `generated-themes`.

I believe the directory created by aggregate translations is hyphenated.

Reply:
I'll follow precedence https://github.com/cerner/terra-toolkit/pull/324/commits/0a976941afbe295532f1f4b742f97462fdcfe2a6